### DISCUSSION TITLE
how to use try/catch   ?

### DISCUSSION THREAD

**Discussion by Hambardzumyan10** [selenide/selenide#2314] [General]
I want to use try/catch in selenide 
but catch not working 

sometimes homepage popup not displaying and for that I use try/catch 

I want when popup is showing click that popup but when not showing click sign in 

          try {
                    mainPage.popup.click();
                } catch (Exception ignored) {
                    mainPage.buttonSign.click();    
                }

**Comment by asolntsev** [comment]
Hi @Hambardzumyan10 !
This is a known issue, described in https://selenide.org/2019/12/02/advent-calendar-how-to-abuse-selenide/
